db/DB.js

```diff
@@ -277,6 +278,9 @@ const DB = {
     })
     .returning("*");
     const updatedSubscriber = Array.isArray(updated) ? updated[0] : null;
+    if (updatedSubscriber) {
+      FXA.destroyOAuthToken({refresh_token: subscriber.fxa_refresh_token});
```

Comment:
Should this be `subscriber.` or `updatedSubscriber.`?

And how is `subscriber.fxa_refresh_token` different from `fxaRefreshToken`? Or is the `subscriber.fxa_refresh_token` just the _old_ token that you want to destroy, I guess...

Reply:
It should be `subscriber` - the *old* subscriber data with the `fxa_refresh_token` that we want to destroy.

Reply:
Does this mean you couldn't be logged into two browsers/sessions at once?
I'm also trying to think of any cases where we'd have orphaned session tokens, or if the logic for "delete all of this subscriber's FxA OAuth tokens _except_ for this latest one, makes any sense.

But yeah, R+ (unless @lesleyjanenorton has any thoughts or objections). Once this lands on dev we can try running a few more scenarios.